docs/docs/spark-procedures.md

```diff
@@ -936,3 +936,40 @@ as an `UPDATE_AFTER` image, resulting in the following pre/post update images:
 |-----|--------|--------------|
 | 3   | Robert | UPDATE_BEFORE|
 | 3   | Dan    | UPDATE_AFTER |
+
+## Table stats
+
+### `compute_table_stats`
+
+This procedure calculates the Number of Distinct Values (NDV) statistics for a specified table.
+By default, statistics are computed for all columns using the table's current snapshot.
+The procedure can be optionally configured to compute statistics for a specific snapshot and/or a subset of columns.
+
+| Argument Name | Required? | Type          | Description                         |
+|---------------|-----------|---------------|-------------------------------------|
+| `table`       | ✔️        | string        | Name of the table                   |
+| `snapshot_id` |           | string        | id of the snapshot to collect stats |
+| `columns`     |           | array<string> | columns to collect stats            |
+
+#### Output
+
+| Output Name       | Type   | Description                                     |
+|-------------------|--------|-------------------------------------------------|
+| `statistics_file` | string | path to stats file created from by this command |
```

Comment:
where is this file located?

Reply:
This is stored in the [metadata directory](https://github.com/apache/iceberg/blob/main/spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/actions/ComputeTableStatsSparkAction.java#L177). Do we want to add this to documentation?

Reply:
I think it will be helpful.

Reply:
Why would we add this to the documentation here? I'm not sure what the value is since it will be evident from the value returned where the file is?

Reply:
I'm not sure this returned value will be visible to users, for example, in a cluster mode Spark application.

Reply:
I also dont understand , the return value will be the location, how is it different in cluster mode?  

I also feel we may not need to document it, the metadata directory is an implementation detail and can be changed at some point.

Reply:
In cluster mode, Spark users can't easily find the returned value. How about adding one line in the description above?

Reply:
This is not a `-1` on the PR, just my preference.

Reply:
I don't have a problem with adding it, but it doesn't belong here. It should be in the method description if we did want to include it.